### DISCUSSION TITLE
Editing/copy-pasting articles.

### DISCUSSION THREAD

**Discussion by cazssiew** [omnivore-app/omnivore#2987] [Q&A] [answered]
Hey guys, I love using omnivore, but I regularly happen upon articles that omnivore fails to parse; I've been looking for a way to edit the text to paste it in myself but there doesn't seem to be such an option. Is there a way to do so that I've missed?

Thanks so much!

**Comment by jacksonh** [comment, marked as answer]
hey yeah this is something we have some ideas on. Unfortunately it could cause a lot of issues if people edited after creating highlights so we haven't implemented yet.

**Reply by Exonymos** [reply]
Are there any updates on this? I would love this feature.
